Update UG for FindCommand

jevan13 opened the pull request:
## Describe your changes
- Update content of FindCommand to reflect latest changes
- Update image for FindCommand example
- Add tips
- Add common mistakes

## Issue ticket number and link
closes #155 

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [X] If it is a core feature, I have added thorough tests.

jevan13 committed:
Update UG for FindCommand

- Update content of FindCommand to reflect latest changes
- Update image for FindCommand example
- Add tips
- Add common mistakes

KiKuasaurus commented on docs/UserGuide.md:
```diff
@@ -483,20 +483,46 @@
+  * `find e p/12345678 e/john@example.com alice@example.com` returns employees with phone number `12345678` and with
+  email either `john@example.com` or `alice@example.com`
+  * `find ph d/IT r/SWE Manager` returns potential hires with department `IT`, and role either `SWE` or `Manager`
+![result for 'find alex david'](images/findjohnResult.png)
```
Should this be [result for `find john`] instead. Should there be another line above it that explains what the image is showing?

jevan13 replied:
Adding a title above! Thanks!

KiKuasaurus replied:
Should the description for when the image does not load also be changed?

jevan13 committed:
Add title for image in UG

- Add title for the example image in UG under FindCommand
- Easy understanding of the image for users

jevan13 committed:
Update image description

- UG for FindCommand
- Image example

KiKuasaurus reviewed (approved):
LGTM!
